.github/workflows/ci_dockers.yml

```diff
@@ -150,3 +150,18 @@ jobs:
           file: dockers/base-conda/Dockerfile
           push: false
         timeout-minutes: 50
+
+  build-nvidia:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      # https://github.com/docker/setup-buildx-action
+      # Set up Docker Buildx - to use cache-from and cache-to argument of buildx command
+      - uses: docker/setup-buildx-action@v1
+      - name: Build CUDA Docker
+        uses: docker/build-push-action@v2
+        with:
+          file: dockers/base-conda/Dockerfile
```

Comment:
Shouldn't it be `dockers/nvidia/Dockerfile` ?

Reply:
good catch, fixed